base/binaryplatforms.jl

```diff
@@ -141,6 +141,7 @@ function Base.setindex!(p::AbstractPlatform, v::String, k::String)
     add_tag!(tags(p), k, v)
     return p
 end
+Base.hash(p::Platform) = hash(hash(a.tags), hash(a.comparison_strategies))
```

Comment:
Should this include `comparison_stratiges` if `==` does not check them?  Or perhaps `==` should include them?

https://github.com/JuliaLang/julia/pull/37734/files#diff-d985676b7739e0f47a51d1f028ec6fedR171

This should also probably include a magic constant so it doesn't hash the same as other data structures.

Reply:
Yeah, `==` should include them.  Thanks, good catch!

Reply:
Actually, my `==` definition doesn't really make sense.  :P

Reply:
Not to mention the fact that I used `p` in the arguments and `a` in the definition.  🙄